### PR TITLE
changed some shit

### PR DESCRIPTION
**Pull request by leetheperm**
broke some stuff for you. How is it looking?

### CODE REVIEW

File: src/Components/App/App.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import { ToDoItems } from "../ToDoItmes/ToDoItems";
 import { ThingsDone } from "../ThingsDone/ThingsDone";
-import './App.css';
+// import './App.css';
 
 class App extends React.Component {
   constructor(props: any) {
     super(props);
-    this.state = {
+    this.state.wrongDudu = {
```
Comment (silviatheweasel) [inline src/Components/App/App.tsx]:
```suggestion
this.state = {
```

File: src/Components/App/App.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { ToDoItems } from "../ToDoItmes/ToDoItems";
 import { ThingsDone } from "../ThingsDone/ThingsDone";
-import './App.css';
+// import './App.css';
```
Comment (silviatheweasel) [inline src/Components/App/App.tsx]: Why have you commented it out? The css file needs to be imported.

File: src/index.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './Components/App/App';
-import reportWebVitals from './reportWebVitals';
+// import reportWebVitals from './reportWebVitals';
```
Comment (silviatheweasel) [inline src/index.tsx]:
```suggestion
import reportWebVitals from './reportWebVitals';
```

**Review by silviatheweasel — CHANGES_REQUESTED** [review]

File: src/Components/App/App.tsx (hunk 2 above)
Comment (silviatheweasel) [inline src/Components/App/App.tsx]:
```suggestion
import './App.css';
```

**Review by leetheperm — COMMENTED** [review]
Yup. made those changes now

**Comment by leetheperm** [comment]
all resolved now.

**Review by leetheperm — COMMENTED** [review]
Done

**Comment by silviatheweasel** [comment]
a bit messed up

**Pull request closed without merging**